src/main/java/org/stellar/sdk/AllowTrustOperation.java

```diff
@@ -94,7 +105,18 @@ public boolean getAuthorize() {
         default:
           throw new RuntimeException("Unknown asset code");
       }
-      authorize = op.getAuthorize();
+
+      int flag = op.getAuthorize().getUint32().intValue();
+      if (flag == TrustLineFlags.AUTHORIZED_FLAG.getValue()) {
```

Comment:
We should probably error on AUTHORIZED_FLAG & AUTHORIZED_TO_MAINTAIN_LIABILITIES_FLAG != 0.